.github/pull_request_template.md

```diff
@@ -7,7 +7,9 @@ Fixes # (issue/s)
 ## Type of change
 
 - [ ] Code cleanup
+- [ ] Unit tests (added or updated unit-tests)
 - [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Enhancement (implementation update, or general performance enhancements)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
\ No newline at end of file
+- [ ] Documentation (documentation-related additions)
```

Comment:
Hi Yanis (@drkameleon),

Do they always have to be additions? Might it be best to leave it at "changes"? Therefore, it would read: "Documentation (documentation-related changes)".

Kind Regards,
Liam

Reply:
@celtic-coder, I do agree! "Changes" sounds better in my opinion.
Sometimes we need to update the documentation usage or remove outdated information, what is not exactly an addition...